codacy/values.yaml

```diff
@@ -31,6 +39,10 @@ global:
     # protocol: "https"
     # disableSSL: "false"
     # isPrivateMode: "true"
+    # app:
+    #  appId: "1234"
+    #  TODO: Add link to documentation
```

Comment:
Remove this and create a task to track that its done when the docs are available